cli/src/trusted_operation.rs

```diff
@@ -87,9 +91,12 @@ pub(crate) fn perform_trusted_operation<T: Decode + Debug>(
 ) -> TrustedOpResult<T> {
 	match top {
 		TrustedOperation::indirect_call(_) => send_indirect_request::<T>(cli, trusted_args, top),
-		TrustedOperation::direct_call(_) => send_direct_request::<T>(cli, trusted_args, top),
 		TrustedOperation::get(getter) =>
 			execute_getter_from_cli_args::<T>(cli, trusted_args, getter),
+		TrustedOperation::direct_call(_) => Err(TrustedOperationError::Default {
+			msg: "Invalid call to `perform_trusted_operation`, use `send_direct_request` directly"
+				.into(),
+		}),
```

Comment:
I decided to remove this method. It is very impractical that all these methods have to return the same type `T`. Our `send_direct_request` only ever returned `()`, which by luck worked as we are trying to decode the `RPCResponse`s value, which was always an empty vector, into it. But this seemed like a massive hack and code smell to me.

Now, we just always return the top-hash.